packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1010,6 +1010,16 @@ async function handler(
           throw new HttpError({ statusCode: 401 });
         }
 
+        // Moving forward in this block is the owner making changes to the booking. All attendees should be affected
+        evt.attendees = originalRescheduledBooking.attendees.map((attendee) => {
+          return {
+            name: attendee.name,
+            email: attendee.email,
+            timeZone: attendee.timeZone,
+            language: { translate: tAttendees, locale: attendee.locale ?? "en" },
+          };
+        });
+
```

Comment:
This is in the block when the organizer reschedules the event. Before this `evt.attendees` only contained a single attendee. This is due to the limitation of our booking page where only one attendee can be entered at a time.